completion_aggregator/api/v1/views.py

```diff
@@ -144,20 +149,21 @@ class CompletionListView(CompletionViewMixin, APIView):
                 }
               ]
             }
-
-    This is a transitional implementation that uses the
-    edx-solutions/progress-edx-platform-extensions models as a backing store.
-    The replacement will have the same interface.
     """
 
     def get(self, request):
         """
         Handler for GET requests.
         """
         paginator = self.pagination_class()  # pylint: disable=not-callable
+        mobile_only = (self.request.query_params.get('mobile_only', 'false')).lower() == 'true'
```

Comment:
Nit: The outer parentheses are unnecessary.

Reply:
Fixed